docs/apis/plugintypes/format/_examples/lib.php

```diff
@@ -0,0 +1,104 @@
+
```

Comment:
It was out of scope. For now, the format class does not have a namespace yet. I developed a new namespaced version during my project week but it will land hopefully at 4.1.